src/main/scala/com/fulcrumgenomics/vcf/HapCutToVcf.scala

```diff
@@ -126,7 +158,12 @@ class HapCutToVcf
       .setOption(Options.INDEX_ON_THE_FLY)
       .setOption(Options.WRITE_FULL_FORMAT_FIELD)
       .setOption(Options.ALLOW_MISSING_FIELDS_IN_HEADER)
-    val writer: VariantContextWriter = builder.build
+    val writer: VariantContextWriter = if (this.createIndex) {
+      builder.setOption(Options.INDEX_ON_THE_FLY).build
```

Comment:
Could we not auto-fill the right answer as true if the output is bgzipped or bcf or (plain VCF but the input has contig lines)?  It sounds, based on the issue, like this is automatable and then we don't have to pass the buck to the user.

Reply:
Buck is now not passed.